pkg/oc/cli/cmd/create/imagestreamtag.go

```diff
@@ -42,105 +41,110 @@ var (
 )
 
 type CreateImageStreamTagOptions struct {
-	ISTag  *imageapi.ImageStreamTag
-	Client imageclient.ImageStreamTagsGetter
-
-	FromImage   string
-	From        string
-	Annotations []string
-
-	DryRun bool
-
-	OutputFormat string
-	Out          io.Writer
-	Printer      ObjectPrinter
+	*CreateSubcommandOptions
+
+	Client imagev1client.ImageStreamTagsGetter
+
+	FromImage          string
+	From               string
+	Annotations        []string
+	Scheduled          bool
+	Insecure           bool
+	Reference          bool
+	ReferencePolicyStr string
+	ReferencePolicy    imagev1.TagReferencePolicyType
 }
 
 // NewCmdCreateImageStreamTag is a command to create a new image stream tag.
-func NewCmdCreateImageStreamTag(name, fullName string, f kcmdutil.Factory, out io.Writer) *cobra.Command {
+func NewCmdCreateImageStreamTag(name, fullName string, f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
 	o := &CreateImageStreamTagOptions{
-		Out: out,
-		ISTag: &imageapi.ImageStreamTag{
-			ObjectMeta: metav1.ObjectMeta{},
-			Tag:        &imageapi.TagReference{},
-		},
+		CreateSubcommandOptions: NewCreateSubcommandOptions(streams),
 	}
-
 	cmd := &cobra.Command{
 		Use:     name + " NAME",
 		Short:   "Create a new image stream tag.",
 		Long:    imageStreamTagLong,
 		Example: fmt.Sprintf(imageStreamTagExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(o.Complete(cmd, f, args))
-			cmdutil.CheckErr(o.Validate())
 			cmdutil.CheckErr(o.Run())
 		},
 		Aliases: []string{"istag"},
 	}
+	cmd.Flags().StringVar(&o.FromImage, "from-image", o.FromImage, "Use the provided remote image with this tag.")
+	cmd.Flags().StringVar(&o.From, "from", o.From, "Use the provided image stream tag or image stream image as the source: [<namespace>/]name[:<tag>|@<id>]")
+	cmd.Flags().StringSliceVarP(&o.Annotations, "annotation", "A", o.Annotations, "Set an annotation on this image stream tag.")
```

Comment:
oh dear.  I wonder how this got in.

Reply:
Will add deprecation message for that shorthand